### PR TITLE
Add some additional debugging to the mother

### PR DESCRIPTION
**Pull request by GeorgeHahn**
Print device id pings as hex
'O' displays ms since seeing a device

Also includes a commit to conform files to root `clang-format` config

### CODE REVIEW

**Comment by ps2** [comment]
There are no changes here to what the app gets for a response to the 'O' command right?

**Comment by GeorgeHahn** [comment]
Correct, this is debug-only

**Comment by GeorgeHahn** [comment]
acab8d3 fixes the empty rows

**Comment by GeorgeHahn** [comment]
Committed against my master branch - I stuck with it because I wanted to work against the previous commits and don't know git well enough to do a nice cherry pick branch without a bit of reading.

File: Shoe/shoe_rom.cpp [hunk 1]
```diff
@@ -19,7 +19,7 @@ uint16_t GetTime()
     // Update timer numbers
     timer.updateTime();
 
-    return (((((timer.t.hours - 5) * 60) + timer.t.minutes) * 60 + timer.t.seconds) / 10) && 0x1FFF;
+    return (((((timer.t.hours - 5) * 60) + timer.t.minutes) * 60 + timer.t.seconds) / 10) & 0x1FFF;
```
Comment (GeorgeHahn) [inline Shoe/shoe_rom.cpp]: 😭 😭 😭

**Pull request merged**